app/api/resources/user.py

```diff
@@ -246,6 +280,10 @@ class LoginUser(Resource):
     @classmethod
     @users_ns.doc('login')
     @users_ns.response(200, 'Successful login', login_response_body_model)
```

Comment:
You can add these to `messages.py` and include here similarly like the other responses i.e `messages.LOGIN_SUCCESS['message']`
Check in other files too where it is hardcoded.

Reply:
done